lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -36,31 +36,35 @@ if mapHasCustomSpawns then
         return cfcRandomSpawn.spawnPointRankings[randomSpawn]["spawn"]
     end
 
-    function cfcRandomSpawn.updateSpawnPointRankings()
-        local averagePlayerDistanceFromSpawns = {}
+    function cfcRandomSpawn.updateSpawnPointRankings( ply )
+        local playerIDSFromSpawns = {}
+	local playerPVPStatus = ply:GetNWBool("CFC_PvP")
 
         for _, spawn in pairs( customSpawnsForMap ) do
-            local playerNetForce = getPlayerForceFromCustomSpawn( spawn )
+			
+	    if playerPVPStatus == spawn["pvp"]		

```

Comment:
I don't think this belongs in this PR?

Reply:
No? The idea is to only consider the set of spawn points which match our pvp status, by skipping over entries tagged with a different pvp status